browser-test/src/admin/admin_program_header.test.ts

```diff
@@ -0,0 +1,29 @@
+import {expect, test} from '../support/civiform_fixtures'
+import {enableFeatureFlag, loginAsAdmin, validateScreenshot} from '../support'
+
+test.describe('admin program page headers', {tag: ['@northstar']}, () => {
+  test('predicate page program header', async ({page, adminPrograms}) => {
+    await enableFeatureFlag(page, 'expanded_form_logic_enabled')
+    await loginAsAdmin(page)
+
+    const programName = 'Predicate page program header'
+    await adminPrograms.addProgram(programName)
+    await adminPrograms.editProgramBlockUsingSpec(programName, {
+      name: 'Screen 1',
+      description: 'first screen',
+      questions: [],
+    })
+
+    await adminPrograms.goToEditBlockEligibilityPredicatePage(
+      programName,
+      'Screen 1',
+      /* expandedFormLogicEnabled= */ true,
+    )
+
+    expect(await page.innerText('#program-title')).toContain(programName)
```

Comment:
```suggestion
    await expect(page.locator('#program-title')).toContainText(programName)
```

(blocking) This is an older playwright style. prefer using web first assertions as they are the recommended format from playwright.

(note) Looking up by id `#program-title` is fine in this case due to the legacyness of that header block. In general avoid getting elements by id or css class. See [playwright docs on using locators](https://playwright.dev/docs/locators#quick-guide) for preferred way of getting elements.